outputs_quantity.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: index
+title: Quantity Outputs
+prev: supply_land.html
+next: outputs_prices.html
+gcam-version: v5.3 
+---
+
+## Description of Outputs
+
+| Name | Resolution | Unit | Query | XML Tag |
+| :--- | :--- | :--- | :--- | :--- |
+| Physical Output<sup>[1](#table_footnote)</sup> | Technology, Region<sup>[2](#table_footnote)</sup>, Vintage, and Year  | Various<sup>[3](#table_footnote)</sup> | <span id="outputs by tech"><button onclick='getQuery("outputs by tech", "outputs by tech")'>outputs by tech</button></span> | `physical-output` |
```

Comment:
currency-output is also specified in startVisitOutput but I don't see it used in queries.